### PR TITLE
NMS-8520: Revert changedBy, keep changedDate

### PR DESCRIPTION
**Pull request by indigo423**
Revert the side effect introduced by fixing the change date.

* JIRA: http://issues.opennms.org/browse/NMS-8520
* Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS884-1

### CODE REVIEW

**Comment by j-white** [comment]
Waiting on confirmation from @indigo423 before merging this.

File: features/ticketing/tsrm-integration/src/main/java/org/opennms/netmgt/ticketer/tsrm/TsrmTicketerPlugin.java [hunk 1]
```diff
@@ -231,6 +229,9 @@ public Ticket get(String ticketId) throws PluginException {
                 if (ownerGroup != null) {
                     ticket.addAttribute(OWNER_GROUP, ownerGroup.getValue());
                 }
+                if (reportedBy != null) {
+                    ticket.setUser(reportedBy.getValue());
+                }
```
Comment (indigo423) [inline features/ticketing/tsrm-integration/src/main/java/org/opennms/netmgt/ticketer/tsrm/TsrmTicketerPlugin.java]: This change doesn't fit to me to solve the problem described in the issue. See here: http://issues.opennms.org/browse/NMS-8520.

**Comment by RangerRick** [comment]
Looking at the code seems obvious enough.  I don't see a bamboo build for it though.  Did it roll off the end?  Was it green?

**Comment by j-white** [comment]
I think it rolled. I re-added it to Bamboo.

**Comment by indigo423** [comment]
Can be merged if bamboo is green.

**Pull request merged**